robottelo/config/validators.py

```diff
@@ -197,6 +197,30 @@
         ),
         Validator("report_portal.fail_threshold", default=20),
     ],
+    repos=[
+        Validator(
+            'repos.rhel6_repo',
+            'repos.rhel7_repo',
+            'repos.rhel8_repo',
+            must_exist=True,
+            is_type_of=str,
+        ),
+        Validator(
+            'repos.rhel6_os',
+            'repos.rhel7_os',
+            'repos.rhel8_os.baseos',
+            'repos.rhel8_os.appstream',
+            'repos.sattools_repo.rhel6',
+            'repos.sattools_repo.rhel7',
+            'repos.sattools_repo.rhel8',
+            'repos.satmaintenance_repo',
+            'repos.rhscl_repo',
+            'repos.ansible_repo',
+            'repos.swid_tools_repo',
+            must_exist=False,
```

Comment:
Why its False ?

Reply:
Sattools repos are one on the important repos of all time.

Reply:
The validator wasn't validating any of these before, and I do not plan on making these fields required with the current validation strategy.

When we move to validation being dynamically executed when relevant tests are collected, then we can make these fields `must_exist`.

Right now, we don't want to make sattools_repo `must_exist` when you might not be running tests that use the field at all.